examples/nextjs-example/src/components/ReactQuery.tsx

```diff
@@ -0,0 +1,12 @@
+"use client";
+
+import { useState } from "react";
+import { QueryClient, QueryClientProvider } from "react-query";
+
+export function ReactQuery({ children }: { children: React.ReactNode }) {
```

Comment:
Why doesn't this just happen in our Provider again?

Reply:
This has nothing to do with the package. This is just used in the example to create a React Query context so we can use it to do the mutation in the example to send the test event. 

I wanted to do an example that didn't use server actions because it's important to check that our react query plays nicely with another one.